spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -197,7 +197,7 @@ $ kubectl create -f src/kubernetes/grafana/
 You can use `kubectl get all -l app=grafana` to verify that the deployment, pod, and service resources are running.
 You can use `kubectl delete all,cm,svc,secrets -l app=grafana` to clean up afterwards.
 
-IMPORTANT: The `url` attribute value shown below should be replaced in `src/kubernetes/server/server-config-rabbit.yaml` or `src/kubernetes/server/server-config-kafka.yaml` to reflect the address and port Grafana is running on. Depending on your setup, the value can obtained by running the command `minikube service --url grafana` if running on Minikube, your load balancer external IP, or DNS name. This configuration is needed for Grafana links to be accessible when accessing the dashboard from a web browser.
+IMPORTANT: The `url` attribute value shown below should be replaced in `src/kubernetes/server/server-config.yaml` to reflect the address and port Grafana is running on. Depending on your setup, the value can obtained by running the command `minikube service --url grafana` if running on Minikube, your load balancer external IP, or DNS name. This configuration is needed for Grafana links to be accessible when accessing the dashboard from a web browser.
```

Comment:
The other inflight PR from Jay rephrased some of this, so there might be conflicts depending on which PR goes in first.

Reply:
given this change is likely much smaller, it would probably be easier to merge this first. /cc @ilayaperumalg